docs/developers/node-tutorial.md

```diff
@@ -119,20 +119,25 @@ an example public Core Endpoint.
   purposes. You can find a list of RPC endpoints [here](../nodes/mamaki-testnet#rpc-endpoints)
 
 ```sh
-celestia light start --core.grpc http://<ip-address>:9090
+celestia light start --core.ip <ip-address> --core.grpc.port <port>
 ```
 
+> NOTE: The `--core.grpc.port` defaults to 9090, so if you do not specify
+  it in the command line, it will default to that port. You can use the flag
+  to specify another port if you prefer.
+
 For example, your command along with an RPC endpoint might look like this:
 
 ```sh
-celestia light start --core.grpc https://rpc-mamaki.pops.one:9090
+celestia light start --core.ip https://rpc-mamaki.pops.one --core.grpc.port 9090
 ```
 
 ### Keys and wallets
 
 You can create your key for your node by running the following command:
 
 ```sh
+celestia light start --core.ip <ip-address> --core.grpc.port <port> --keyring.accname developer
```

Comment:
?

Reply:
I think something happened to the old key instruction docs?

Reply:
@renaynay can you provide me context on what happened?

Reply:
Like, what changes are you specifically requesting here?

Reply:
The command is to start a light node, not to create a key